environments/dbsql-nyc-taxi-trip-analysis/main.tf

```diff
@@ -0,0 +1,25 @@
+terraform {
+  required_providers {
+    databricks = {
+      source  = "databricks/databricks"
+      version = ">= 1.0.0"
+    }
+  }
+}
+
+provider "databricks" {
+  # For an overview of possible authentication mechanisms, please refer to:
+  # https://registry.terraform.io/providers/databricks/databricks/latest/docs#authentication
+}
+
+data "databricks_group" "users" {
+  display_name = "users"
+}
+
+resource "databricks_sql_endpoint" "this" {
+  name         = "Sample endpoint"
```

Comment:
average user we target this example towards most likely has no access to create an endpoint. Please provide a variable to use existing endpoint id

Reply:
It's not the endpoint ID we need, we need the attribute of an endpoint which is the data source ID.

Could be a TF data source itself, but it doesn't exist today.